cfme/fixtures/v2v_fixtures.py

```diff
@@ -29,6 +30,29 @@
 V2vProviders = namedtuple("V2vProviders", ["vmware_provider", "rhv_provider", "osp_provider"])
 
 
+def set_skip_event_history_flag(appliance):
+    """This flag is required for OSP to skip all old events and refresh inventory"""
+    appliance.update_advanced_settings(
+        {'ems': {'ems_openstack': {'event_handling': {'event_skip_history': True}}}})
+    appliance.evmserverd.restart()
+    appliance.evmserverd.wait_for_running()
+    appliance.wait_for_web_ui()
+
+
+def start_event_workers_for_osp(appliance, provider):
+    """This is a workaround to start event catchers until BZ 1753364 is fixed"""
+    provider_edit_view = navigate_to(provider, 'Edit')
+    endpoint_view = provider.endpoints_form(parent=provider_edit_view)
+    endpoint_view.events.event_stream.select("AMQP")
+    endpoint_view.events.event_stream.select("Ceilometer")
+    provider_edit_view.save.click()
+    # Test if workers are enabled and running
+    worker_status = appliance.ssh_client.run_rake_command(
+        "evm:status | grep 'Openstack::Cloud::EventCatcher'").success
```

Comment:
**Suggestion:** 
Checking another column for `Openstack::Cloud::EventCatcher` `status` as `Started` or `Stopped` in the `.output` would be great for further test.

Like this: 

```suggestion
     worker_status = appliance.ssh_client.run_rake_command(
        "evm:status | grep 'Openstack::Cloud::EventCatcher'")
     assert "started" in workder_status.output
     if not worker_status.success:
```

Reply:
Hi Satya.. The command returns 'True' on success and I am checking if the status is True

Reply:
@sshveta Yes, I am agree with you. `.success` will return `True` as worker entry shown in the output of `bin/rake evm:status`. But worker status might be `Started/Stopped/Stop Pending/Error`.
If you're not checking `status` of `worker` then your test which is using this function may fail.